src/database/migrations/2024_03_27_150233_add_selected_duplicate_algo_to_crews.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('crews', function (Blueprint $table) {
+            $table->foreignUuid('duplicate_algorithm_id')->nullable();
```

Comment:
Let's not this be nullable. 
The crew must have a duplicate_algorithm_id : either the one selected by the user / else it should be the default algo